.github/workflows/live-tests.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           PROOF_LEVEL: 'none'
         ports:
           - 3085:3085
+          - 5432:5432
           - 8080:8080
           - 8181:8181
```

Comment:
out of curiosity, what are these different ports for?

Reply:
Some of them are not used as of now and here just for example to others.

- PostgreSQL one, 5432, since we might want to also run more complex scenarios later, not in use currently;
- 8080, Nginx reverse proxy in order to properly configure CORS settings;
- Mina Accounts-Manager one, 8181, currently is not used but will be used after https://github.com/o1-labs/quality-and-testing/issues/121;
- Mina Daemon default GraphQL port, 3085, not used at all, stays here just for example.

More docs [here](https://hub.docker.com/r/o1labs/mina-local-network).

Reply:
@mitschabaude good to merge?